worksheet_B/readme.MD

```diff
@@ -0,0 +1,38 @@
+code:
+```
+
+START
+
+Attempts=0
+i=0
+Likeness=0
+OriginalWord=Generate
+
+while (attempts <= 4):
+	Output ("enter your first attempt: ")
+	UserEntery=UserInput
+	attempts+1
+	
+	For (each character in OriginalWord):
+		i+1
+		if (letter == letter in UserEntery)
+			Likeness+1
+		else
+			next character
+		END IF
+	Output ("your likeness is: " + Likeness)
+	END FOR
+END WHILE
+
+if (Likeness == 4)
+	output ("congratulations")
+else
+	output ("you have failed")
+END IF
+
+END
+
+```
+
+
+![Flowchart](flow.png)
```

Comment:
Worksheet C (attempt.docx):

(a) Explanation is unclear, but I think you got the idea that the algorithm checks for duplicate items.

(b) O is not a variable, it is a piece of notation -- O(...) basically just means "grows like ... when the input is large".

(c) Not sure what you mean here.